test/runtests.jl

```diff
@@ -16,36 +16,48 @@ A = [ 1.7     0     0     0     0     0     0     0   .13     0
 A = sparse(A)
 
 for perm ∈ (collect(1 : A.n), amd(A))
-  L, d, α, P = lldl(A, perm, memory=0)
-  nnzl0 = nnz(L)
+  LLDL = lldl(A, perm, memory=0)
+  nnzl0 = nnz(LLDL.L)
```

Comment:
Maybe we could define `nnz` on objects of type `LimitedLDLFactorization`?

Reply:
Yes, good idea.